.gitignore

```diff
@@ -3,5 +3,8 @@
 *.pyc
 
 # Generated during a regular build
-build
+html
 source/locale/pot/.doctrees
+
+# VSCode generated files
+.vscode
```

Comment:
Please don't add it here. We cannot add entries for every possible IDE/editor out there. Add it to you global gitingore instand: https://openqa-bites.github.io/posts/2021-04-23-global-gitignore/